src/styles.scss

```
@@ -164,6 +164,13 @@ footer.docs {
   width: calc(100% - 280px);
 }
 
+blockquote.warning {
```

I would suggest we apply this to any tag with a warning class.

```suggestion
.warning {
```

---

4d5ffcf